projects/demo/src/styles.less

```diff
@@ -1,12 +1,14 @@
 @import '@taiga-ui/core/styles/taiga-ui-local';
 
-html,
-body {
-    .scroll-behavior();
-    font: var(--tui-font-text-m);
+html[data-tui-theme] {
```

Comment:
These do not matter, it is just our demo app.

Reply:
fixed